ansible/deploy.yml

```diff
@@ -33,3 +33,17 @@
       include: render_and_copy_to_s3.yml file={{item}} permission="public-read"
       with_items:
         - configuration/configuration.json
+
+    - name: create EFS volume for persistent home dirs   
+      efs:
+        state: present
+        name: ATMO-prod 
+        tags:
+            app: telemetry-analysis
+        targets:
+            - subnet_id: subnet-63f72706
+              security_groups: [ "sg-0964dc71" ]
```

Comment:
I think we should create this security group via ansible using the ec2_group module [1]. The output should return the security group id and can be referenced in the efs creation. 

[1] http://docs.ansible.com/ansible/ec2_group_module.html